consensus/node.go

```diff
@@ -103,6 +106,18 @@ func (n *Node) Stop() error {
 		<-n.tendermint.Quit()
 		n.logger.Info("Tendermint node stopped")
 	}
+	
+	for ctxID, db := range n.dbs {
```

Comment:
shouldn't this happen only if tendermint.isRunning() ? inside the if condition above?

Reply:
Not really, because even if the Tendemint node is closed the dbs might remain open and that is the reason to keep it outside

Reply:
yea I was referring more to the case where it never started and thinking if attempting to close a not open DB would throw an error.